src/Typesense/SearchParameters.cs

```diff
@@ -269,6 +269,12 @@ public record SearchParameters
     [JsonPropertyName("highlight_full_fields")]
     public string? HighlightFullFields { get; set; }
 
+    /// <summary>
+    /// List of fields which should be highlighted with snippetting.
+    /// </summary>
```

Comment:
Add more documentation to the documentation, you could for example use the documentation from the API spec. https://typesense.org/docs/27.0/api/search.html#results-parameters